.github/workflows/lint.yml

```diff
@@ -0,0 +1,45 @@
+name: Lint
+
+on:
+  push:
+    branches:
+    - stable
+    - devel
+    tags:
+    - '[0-9]+.[0-9]+.[0-9]+'
+    - '[0-9]+.[0-9]+.[0-9]+a[0-9]+'
+  pull_request:
+    branches:
+    - stable
+    - devel
+
+jobs:
+  lint:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest]
+        python-version: ['3.10']
+    timeout-minutes: 60
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
```

Comment:
Maybe add `cache: 'pip'` as well to make it a bit faster? See [here](https://github.blog/changelog/2021-11-23-github-actions-setup-python-now-supports-dependency-caching/).

Reply:
I'm actually not worried about the time it takes to install dependencies but the additional nodes that are required for a separate safety workflow.

Reply:
If the nodes would run for only half the time that would be fine though. But that wouldn't happen here since the time is spent in the other tox workflows and tox does not support caching.

Reply:
But yeah, it's fine for now I think.